### DISCUSSION TITLE
Responsive primitive navigationmenu with RadixUI theme

### DISCUSSION THREAD

**Discussion by Saadchr** [radix-ui/themes#176] [Q&A]
Has anyone a exemple of a responsive implementation of a primitive navigationmenu with RadixUI theme ?

**Comment by vladmoroz** [comment]
Do you have an example design in mind?

**Reply by Saadchr** [reply]
I'm thinking about the [https://www.apple.com/](https://www.apple.com/) navigation menu.
Webstudio.is has done a video showing the implementation. I think they're using radixUI primitive or the shadcnUI implementation
https://www.youtube.com/watch?v=1xSrvEkXWws&t=318s

**Reply by jonschmidt** [reply]
Would also be interested in this. We have almost the identical menu built using radix UI and we are not that happy with the implementation, but we have not had the time to revisit.
